superset-frontend/src/explore/components/controls/DndColumnSelectControl/DndMetricSelect.tsx

```diff
@@ -125,7 +126,7 @@ const getMetricsMatchingCurrentDataset = (
   }, []);
 };
 
-export const DndMetricSelect = (props: any) => {
+const DndMetricSelect = (props: any) => {
```

Comment:
We definitely should improve typing here... (not in this PR, that's a general note)